src/admin-views/aggregator/header.php

```diff
@@ -0,0 +1,6 @@
+<h1><?php echo esc_html( $this->get_page_title() ); ?></h1>
+<h2 class="nav-tab-wrapper">
+	<?php foreach ( $this->tabs->get() as $tab ): ?>
+	<a id="<?php echo esc_attr( $tab->get_slug() ); ?>" class="nav-tab<?php echo ( $tab->is_active() ? ' nav-tab-active' : '' ); ?>" href="<?php echo $tab->get_url(); ?>"><?php echo $tab->get_label(); ?></a>
```

Comment:
Move the `esc_url()` and `esc_html()` calls from the `get_url()` and `get_label()` methods and do the escaping here instead?